eth/v1alpha1/validator.proto

```diff
@@ -312,6 +322,22 @@ message ValidatorStatusResponse {
     uint64 position_in_activation_queue = 5;
 }
 
+message MultipleValidatorStatusRequest {
+    // A list of 48 byte validator public keys.
+    repeated bytes public_keys = 1 [(gogoproto.moretags) = "ssz-size:\"?,48\""];
+    // A list of validator indices.
+    repeated int64 indices = 2;
+}
+
+message MultipleValidatorStatusResponse {
+    // A list of 48 byte validator public keys.
+    repeated bytes public_keys = 1 [(gogoproto.moretags) = "ssz-size:\"?,48\""];
+    // A list of ValidatorStatusResponses mapped 1-to-1 with the public keys.
+    repeated ValidatorStatusResponse statuses = 2;
+    // A list of indices in the request where the validator status could not be determined.
+    repeated int64 missing_indices = 3;
```

Comment:
Is this necessary? What about missing public keys? I am not sure this is a good pattern to follow

Reply:
Missing public keys gets an `UNKNOWN STATUS` by default. But yeah, we could just skip them.

Reply:
Okay. It is removed.